compiler/test/data/typescript/mergeDeclarations/namespaceWith/class.d.kt

```diff
@@ -0,0 +1,43 @@
+// [test] nonDeclarations.lunr.nonDeclarations.kt
+@file:Suppress("INTERFACE_WITH_SUPERCLASS", "OVERRIDING_FINAL_MEMBER", "RETURN_TYPE_MISMATCH_ON_OVERRIDE", "CONFLICTING_OVERLOADS")
+package lunr
+
+import kotlin.js.*
+import kotlin.js.Json
+import org.khronos.webgl.*
+import org.w3c.dom.*
+import org.w3c.dom.events.*
+import org.w3c.dom.parsing.*
+import org.w3c.dom.svg.*
+import org.w3c.dom.url.*
+import org.w3c.fetch.*
+import org.w3c.files.*
+import org.w3c.notifications.*
+import org.w3c.performance.*
+import org.w3c.workers.*
+import org.w3c.xhr.*
+
+typealias Plugin = (self: Builder, args: Any) -> Unit
+
+// ------------------------------------------------------------------------------------------
+// [test] class.lunr.kt
+@file:JsQualifier("lunr")
```

Comment:
if it is a name of actually existing package I'd rather rename

Reply:
Sure